repo/packages/K/kubernetes/1/marathon.json

```diff
@@ -0,0 +1,91 @@
+{
+  "id": "{{kubernetes.app-id}}",
```

Comment:
we are moving services to use the framework-name here.   The independent app-id causes confusion and breaks some of the current assumptions in dcos